doc/manuals/work_queue/index.md

```diff
@@ -1222,6 +1222,56 @@ API](http://ccl.cse.nd.edu/software/manuals/api/html/work__queue_8h.html).
 
 ### Security
 
+#### TLS (SSL) support
+
+Work Queue can encrypt the communication between manager and workers using TLS.
+For this, you need to specify the key and certificate (in PEM format) of your
+server when creating the queue.
+
+If you do not have a key and certificate at hand, but you want the
+communications to be encrypted, you can create them using a command like:
+
```

Comment:
Explain a little bit more here that this is self-signed, with the usual proviso...